book/views.py

```diff
@@ -118,4 +118,68 @@ def get(self, request):
             return JsonResponse({"message":"SUCCESS", "books":json_data}, status=200)
 
         return JsonResponse({"message":"INVALID_REQUEST"}, status=400)
-        
+
+class ReviewView(View):
+    def post(self, request, book_id):
+        data = json.loads(request.body)
+        try :
+            user_id  = data['user_id']
+            contents = data['contents']
+
+            if len(contents) < 200:
+                review = Review.objects.create(
+                    user_id  = user_id,
+                    book_id  = book_id,
+                    contents = contents
+                )
+                return JsonResponse({'message':'SUCCESS'}, status=200)
+            return JsonResponse({'message':'LONG_CONTENTS'}, status=400)
+        except KeyError:
+            return JsonResponse({'message':'KEY_ERROR'}, status=400)
+
+    def get(self, request, book_id):
+        data = json.loads(request.body)
+        try:
+            user_id = data['user_id']
+            reviews = Review.objects.select_related('user').filter(user_id=user_id)
+            review_list = [{
+                'review_id'  : review.id,
+                'nick_name'  : review.user.nickname,
+                'user_img'   : review.user.image_url,
+                'content'    : review.contents,
+                'created_at' : review.created_at.strftime('%Y.%m.%d'),
+            } for review in reviews ]
+            return JsonResponse({'review_list':review_list}, status=200)
+        except Review.DoesNotExist:
+            return JsonResponse({'message':'NOT_EXIST_REVIEW'}, status=400)
+
+    def delete(self, request, book_id):
+        data = json.loads(request.body)
+        try :
+            user_id = data['user_id']
+            review_id = request.GET['review_id']
+            review  = Review.objects.get(id=review_id)
+            if review.user_id == user_id:
+                review.delete()
+                return JsonResponse({'meassage':'SUCCESS'}, status=200)
+            return JsonResponse({'meassage':'NOT_THIS_USER'}, status=400)
```

Comment:
```suggestion
            return JsonResponse({'meassage':'UNAUTHORIZED'}, status=400)
```
내 리뷰가 아닌 다른 사람의 리뷰를 지우는 과정인 것이죠? 그럴 권한이 없다는 의미로 이런 메세지를 주면 좋을 것 같습니다.

Reply:
감사합니다 수정하였습니다~~